packages/backend/src/index.ts

```diff
@@ -202,6 +203,13 @@ async function main() {
     router: jenkins,
     isOptional: true,
   });
+  await addPlugin({
+    plugin: 'permission',
+    config,
+    apiRouter,
+    createEnv,
+    router: permission,
+  });
```

Comment:
```suggestion
  await addPlugin({
    plugin: 'permission',
    config,
    apiRouter,
    createEnv,
    router: permission,
    isOptional: true,
  });
```
The plugin should have `isOptional: true`.